<a href="https://colab.research.google.com/github/AinsleyJacquemain/ASTRON-1221/blob/main/Calculating_the_Age_of_the_Universe_from_the_Hubble_Constant.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [2]:
import astropy.units as u # import a Python package to call certain units.
import astropy.constants as const # import a Python package to call the value of certain constants.
import numpy as np  # import numpy package, which allows us to conduct mathematical operations for the defined quantities.

In [4]:
d = 1 * u.Mpc # define d, or the distance.
print(d)

1.0 Mpc


In [7]:
v = 70 * u.km/u.s # define v, or the velocity, which is, in this instance, the Hubble Constant.
print(v.to(u.Mpc / u.s))

2.2685455026110553e-18 Mpc / s


In [23]:
def CalculatingUniverseAge(d, v): # d and v represent the input of the function.
    t = np.divide(d, v)
    return t
print(t.to(u.Gyr)) # define the output, which is the function for t, or the time.

13.968460309725558 Gyr


The age of the universe based on the Hubble Constant calculation is approximately 13.97 Gyr, or approximately 1.397 * 10^10  years.

